src/mame/machine/elan_ep3a19asys.cpp

```diff
@@ -0,0 +1,117 @@
+// license:BSD-3-Clause
+// copyright-holders:David Haywood
+
+#include "emu.h"
+#include "elan_ep3a19asys.h"
+
+// like EP3A19A but with the DMA at a lower address where the code bank register would usually be?
```

Comment:
Did you mean to say "like EU3A05 but..." here?

Reply:
ah yes, that's what happens when you search+replace